lib/json_spec/cucumber.rb

```diff
@@ -18,6 +18,18 @@
   end
 end
 
+Then /^the (?:JSON|json)(?: response)?(?: at "(.*)")? with UUIDs should( not)? be:$/ do |path, negative, json|
```

Comment:
also no mention of time apart from hidden away in the code does not seem so good